app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -60,19 +60,22 @@ class SecureMessageController @Inject()(
               .createConversation(conversation)
               .map {
                 case Right(_) =>
-                  val _ = auditCreateConversation(EventTypes.Succeeded, conversation)
+                  val _ = auditCreateConversation(EventTypes.Succeeded, conversation, "Conversation Created")
                   Created
                 case Left(error: SecureMessageError) =>
-                  val _ = auditCreateConversation(EventTypes.Failed, conversation)
+                  val _ = auditCreateConversation(EventTypes.Failed, conversation, error.message)
```

Comment:
You don't need to call auditing here as our error handler calls bootstrap error handler which logs and audits the events: https://github.com/hmrc/secure-message/blob/master/app/uk/gov/hmrc/securemessage/ErrorHandler.scala#L32

Reply:
okay, I'll take all of this out

Reply:
Will this be logged in Splunk explicitly then?

Reply:
Yes, look at the implementation of the error logger from hmrc library we call in our error handler, they call both audit and log